### PR TITLE
BUGFIX: Get price ID of product with variable prices included in Bundle

### PR DESCRIPTION
**Pull request by sergeykomlev**
For product(s) with variable prices included in Bundle(s) we need to get price ID in a different way. Without this change, customers see all downloadable files for variable pricing products included in the Bundle.

### CODE REVIEW

File: templates/history-downloads.php [hunk 1]
```diff
@@ -36,6 +36,10 @@
 					<tr class="edd_download_history_row">
 						<?php
 						$price_id       = edd_get_cart_item_price_id( $download );
+						// Get price ID of product with variable prices included in Bundle
+						if ( $download['in_bundle'] && edd_has_variable_prices( $download['id'] ) ) {
```
Comment (robincornett) [inline templates/history-downloads.php:40]:
```suggestion
						if ( ! empty( $download['in_bundle'] ) && edd_has_variable_prices( $download['id'] ) ) {
```
I think we need to add the `empty` check as the `in_bundle` may not be set at all--I got an undefined index notice.

Once I made that change, the PR works as expected and does prevent all bundled items from showing for the download if they should not be.

**Review by robincornett — CHANGES_REQUESTED** [review]

File: templates/history-downloads.php (hunk 1 above)
Comment (sergeykomlev) [reply templates/history-downloads.php:40]: @robincornett, Thanks! Great suggestion.

**Review by sergeykomlev — COMMENTED** [review]
Great suggestion!

**Review by robincornett — APPROVED** [review]
Thanks for the PR, @sergeykomlev!

**Comment by sergeykomlev** [comment]
Hi @ashleyfae, will this change will be applied in the next releases?

**Review by cklosowski — APPROVED** [review]

**Pull request merged**